drv/cosmo-seq-server/src/main.rs

```diff
@@ -664,6 +676,35 @@ impl NotificationHandler for ServerImpl {
             _ => (),
         }
 
+        // If we're powered up, detect powering down
```

Comment:
Minor point, but reset conditions in this case don't represent powering down, but rather, detection of the CPU spontaneously resetting itself.

Reply:
Clarified in 9aa7d90bffb9